doc/Compilation.md

```diff
@@ -308,33 +320,33 @@ architectures.
 
 To leverage the Tock build system, you must:
 
-  1. Set `TOCK_USERLAND_BASE_DIR` to the path to the Tock userland
-  2. `include $(TOCK_USERLAND_BASE_DIR)/AppMakefile.mk`
+  1. Set `TOCK_USERLAND_BASE_DIR` to the path to the Tock userland.
+  2. `include $(TOCK_USERLAND_BASE_DIR)/AppMakefile.mk`.
 
 This `include` should be the _last_ line of the Makefile for most applications.
 
 In addition, you must specify the sources for your application:
 
-  - `C_SRCS` - A list of C files to compile
-  - `CXX_SRCS` - A list of C++ files to compile
-  - `AS_SRCS` - A list of assembly files to compile
-  - `EXTERN_LIBS` - A list of directories for libraries [**compiled for Tock**](#compiling-libraries-for-tock)
+  - `C_SRCS`: A list of C files to compile.
+  - `CXX_SRCS`: A list of C++ files to compile.
+  - `AS_SRCS`: A list of assembly files to compile.
+  - `EXTERN_LIBS`: A list of directories for libraries [**compiled for Tock**](#compiling-libraries-for-tock).
```

Comment:
What's the motivation for the change from `:` to `-`? I feel like the `:` attached to the padded teletype looks weird:

![image](https://user-images.githubusercontent.com/339422/28549407-cba0fb5a-70a7-11e7-9036-b64e5746a9fb.png)